test/interface/linear_nonlinear_tests.jl

```diff
@@ -44,28 +44,28 @@ solref = @test_nowarn solve(prob,
     TRBDF2(autodiff = AutoFiniteDiff(), linsolve = KrylovJL_GMRES(),
         smooth_est = false));
 @test length(sol.t) < 20
-sol = @test_nowarn solve(prob,
+sol = solve(prob,
     TRBDF2(autodiff = AutoFiniteDiff(), linsolve = KrylovJL_GMRES(),
         precs = precsl, smooth_est = false, concrete_jac = true));
```

Comment:
instead update the preconditioners being tested? Also the docs tutorial on this will need an update.